src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section
+    class="ec-table-filter"
+    data-test="ec-table-filter"
+  >
+    <component
+      :is="filter.component"
+      v-for="(filter, index) in filters"
+      :key="index"
+      :value="value[filter.name]"
+      v-bind="filter"
+      :label="filter.label"
+      :data-test="`ec-table-filter__filter-item-${index}`"
+      class="ec-table-filter__filter-item"
+      @change="onChange(filter.name, $event)"
+    />
+    <!-- TODO ONL-4893
+    <button
+      type="button"
+      data-test="ec-table-filter__less-filters-button"
+      class="ec-table-filter__less-filters-button"
+      @click="handleLessFilterClick()"
+    >
+      {{ setLessFiltersButtonText }}
+      <ec-icon
+        class="ec-table-filter__less-filters-button--icon"
+        :size="16"
+        :name="setLessFiltersButtonIcon"
```

Comment:
`toggleMoreFiltersButtonIcon`. `set...` means "something will be changed to a given value". Getting a name should not be setting anything.